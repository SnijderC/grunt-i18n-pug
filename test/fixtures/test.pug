doctype html
html
    head
        meta(charset="utf-8")
        title Test page
    body
        h1 Localised versions of #[code test.pug]
        p=__("What would you say if I gave you {num} %s apple?", "What would you say if I gave you {num} %s apples?", apples.type, apples)
        p=__("What would you say if I gave you {num} {type} pears?", "What would you say if I gave you {num} {type} pears?", pears)
