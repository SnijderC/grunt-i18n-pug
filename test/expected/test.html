<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Test page</title>
  </head>
  <body>
    <h1>Localised versions of <code>test.pug</code></h1>
    <p>What would you say if I gave you 1 tasty apple?</p>
    <p>What would you say if I gave you 5 spoiled pears?</p>
  </body>
</html>
